src/modules/75-cf/pages/onboarding/__test__/OnboardingDetailPage.test.tsx

```diff
@@ -244,4 +258,27 @@ describe('OnboardingDetailPage', () => {
 
     expect(container).toMatchSnapshot()
   })
+
+  test('should navigate to the previous page if you click the [x] icon and preference exists', async () => {
+    renderComponent()
+    userEvent.click(screen.getByTestId('close'))
+
+    expect(screen.findByTestId('location'))
+    expect(screen.getByTestId('location')).toHaveTextContent(preferenceURL)
```

Comment:
You can ditch the first `expect` as it isn't actually doing anything

```suggestion
    expect(screen.getByTestId('location')).toHaveTextContent(preferenceURL)
```

Reply:
removed :)